app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  validates :first_name, presence: true, length: { maximum: 25 }
+  validates :second_name, presence: true, length: { maximum: 25 }
+
+  before_save { self.email = email.downcase }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
```

Comment:
On line one you have `# frozen_string_literal: true`. Do you need to add `.freeze` here as well?

Reply:
Ive removed the `# frozen_string_literal: true` and kept the `.freeze`.

When I removed the `.freeze` rubocop gave a warning, so I decided to keep that one instead.